terraform/files/bin/deploy_cluster_api.sh

```diff
@@ -37,6 +37,8 @@ cp -p $HOME/cluster-defaults/clusterctl.yaml $HOME/.cluster-api/clusterctl.yaml
 echo "deploy cluster-api with openstack provider $CLUSTERAPI_OPENSTACK_PROVIDER_VERSION"
 clusterctl init --infrastructure openstack:v$CLUSTERAPI_OPENSTACK_PROVIDER_VERSION --core cluster-api:v$CLUSTERAPI_VERSION -b kubeadm:v$CLUSTERAPI_VERSION -c kubeadm:v$CLUSTERAPI_VERSION
 
+kubectl -n capo-system set env deployment/capo-controller-manager CLUSTER_API_OPENSTACK_INSTANCE_CREATE_TIMEOUT=$CLUSTER_API_OPENSTACK_INSTANCE_CREATE_TIMEOUT
```

Comment:
This looks exactly like what I did manually. I simply added the environment variable to the container spec, which is defined in this deployment.

Reply:
Thank you for your information. That was what I expected. AFAIK the `clusterctl init` command doesn't provide adding environment variables to the provider operator, so that should do the trick for now.